src/e2e-test/features/mssql/RunTime.feature

```diff
@@ -0,0 +1,81 @@
+    And Click on the Next button in Replication mode "Next"
+    Then Validate Source table is available and select it
+    And Click on the Next button in Replication mode "Next"
+    Then Wait till the Configure Advanced Properties page is loaded in replication
+    And Click on the Next button in Replication mode "Next"
+    Then Wait till the Review Assessment page is loaded in replication
+    And Click on the Next button in Replication mode "Next"
+    Then Deploy the replication pipeline
+    And Run the replication Pipeline
+    Then Open the Advanced logs
+    And Wait till pipeline is in running state and check if no errors occurred
+#    Then Verify expected MsSQL records in target BigQuery table
+    And Open and capture logs
+    Then Close the replication pipeline logs and stop the pipeline
+
+  @MSSQL_SOURCE @MSSQL_DELETE @BQ_SINK_TEST
+  Scenario: To verify replication of snapshot and cdc data from MsSQL to BigQuery successfully
+    Given Open DataFusion Project with replication to configure pipeline
+    When Enter input plugin property: "name" with pipelineName
+    And Click on the Next button in Replication mode "Next"
+    And Select Source plugin: "Microsoft SQLServer" from the replication plugins list
+    Then Replace input plugin property: "host" with value: "mssqlHost" for Credentials and Authorization related fields
+    Then Replace input plugin property: "port" with value: "mssqlPort" for Credentials and Authorization related fields
+    Then Select dropdown plugin property: "select-jdbcPluginName" with option value: "mssqlDriverName"
+    Then Replace input plugin property: "database" with value: "mssqlDatabaseName"
+    Then Replace input plugin property: "user" with value: "mssqlUsername" for Credentials and Authorization related fields
+    Then Replace input plugin property: "password" with value: "mssqlPassword" for Credentials and Authorization related fields
+    And Click on the Next button in Replication mode "Next"
+    Then Replace input plugin property: "project" with value: "projectId"
+    Then Enter input plugin property: "datasetName" with value: "dataset"
+    And Click on the Next button in Replication mode "Next"
+    Then Validate Source table is available and select it
+    And Click on the Next button in Replication mode "Next"
+    Then Wait till the Configure Advanced Properties page is loaded in replication
+    And Click on the Next button in Replication mode "Next"
+    Then Wait till the Review Assessment page is loaded in replication
+    And Click on the Next button in Replication mode "Next"
+    Then Deploy the replication pipeline
+    And Run the replication Pipeline
+    Then Open the Advanced logs
+    And Wait till pipeline is in running state and check if no errors occurred
+#    Then Verify expected MsSQL records in target BigQuery table
+    And Run insert, update and delete CDC events on source table
+    And Wait till CDC events are reflected in BQ
```

Comment:
I believe it would make more sense to add a wait before starting the validation instead of adding a step.

Reply:
Here we are waiting for a load interval already.